pos_receipt_replace_user_by_trigram/static/src/xml/order_receipt.xml

```diff
@@ -0,0 +1,11 @@
+<templates id="template" xml:space="preserve">
+    <t t-inherit="point_of_sale.OrderReceipt" t-inherit-mode="extension" owl="1">
+        <xpath expr="//div[hasclass('cashier')]" position="after">
```

Comment:
Hi, it appends.So I just pushed another change. I think it would make more sense when 
The module `pos_receipt_replace_user_by_trigram` depends on `pos_receipt_hide_user`